src/com/company/BankAccount.java

```diff
@@ -1,37 +1,29 @@
 package com.company;
 
 public class BankAccount {
-    private int number;
+    private final int number;
     private double balance;
-    private String currency;
+    private final String currency;
```

Comment:
Order, first all finals, then event blank line (not necessarily), then non-final